### PR TITLE
feat(is-valid-pj): adds validation for processo juridico

### PR DESCRIPTION
**Pull request by felipediogo**

### CODE REVIEW

File: src/utilities/processo-juridico/index.ts [hunk 1]
```diff
@@ -21,3 +21,80 @@ export function format(processoJuridico: string) {
...
+  }
+
+  const value = processoJuridico.split('');
+  const digit = value.splice(7, 2).join('');
+
+  let procJudNum = value.join('');
+  procJudNum = '00000000000000000' + procJudNum;
+  procJudNum = procJudNum.slice(procJudNum.length - 18);
+
+  let b1, b2, b3, b4, b5, b6, b7, b8, b9, b10, b11, b12, b13, b14, b15, b16, b17, b18;
```
Comment (hyanmandian) [inline src/utilities/processo-juridico/index.ts:37]: I think to make your logic easier to understand, you should split the process into functions. What do you think about it?

**Comment by hyanmandian** [comment]
Please, if you have time, don't forget to add some docs to this feature. I think it can help other people to use it in the future. Just edit the pt and en files inside the docs directory.

File: src/utilities/processo-juridico/index.ts (hunk 1 above)
Comment (felipediogo) [reply src/utilities/processo-juridico/index.ts:37]: Yeah, it makes sense. I don't understand too much about the mathematical process behind it. But I'll see what I can do.

Comment (andreoav) [reply src/utilities/processo-juridico/index.ts:37]: Can we translate `processoJuridico` to english? I know the lib is `brazilian` utils, but we keep source code in english.

**Pull request closed without merging**